src/Stratis.Bitcoin.IntegrationTests/APITests.cs

```diff
@@ -132,26 +104,22 @@ public void CanStartStakingViaAPI()
         [Fact]
         public void CanCallRPCMethodViaRPCsCallByNameAPI()
         {
-            try
+            using (NodeBuilder builder = NodeBuilder.Create())
             {
-                var fullNode = this.apiTestsFixture.stratisPowNode.FullNode;
-                var apiURI = fullNode.NodeService<ApiSettings>().ApiUri;
+                var posNode = builder.CreateStratisPosApiNode();
+                posNode.Start();
```

Comment:
Maybe reduce a line - pass in startparam as true ? `var posNode = builder.CreateStratisPosApiNode(true)`

Reply:
Fixed